streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/FlinkClusterServiceImpl.java

```diff
@@ -160,16 +165,19 @@ public void start(FlinkCluster cluster) {
       if (ExecutionMode.isYarnSessionMode(flinkCluster.getExecutionModeEnum())) {
         String address = YarnUtils.getRMWebAppURL() + "/proxy/" + deployResponse.clusterId() + "/";
         flinkCluster.setAddress(address);
+        flinkCluster.setJobManagerUrl(deployResponse.address());
       } else {
         flinkCluster.setAddress(deployResponse.address());
       }
       flinkCluster.setClusterId(deployResponse.clusterId());
-      flinkCluster.setClusterState(ClusterState.STARTED.getValue());
+      flinkCluster.setClusterState(ClusterState.RUNNING.getValue());
       flinkCluster.setException(null);
       updateById(flinkCluster);
       FlinkRESTAPIWatcher.removeFlinkCluster(flinkCluster);
```

Comment:
if the cluster is started successfully here, then we should add it into the monitor ?

In other scenerios with my limited read , `update`, `delete`,`shutdown` `start`, Its should be processed the logic by using FlinkClusterWatcher.addFlinkCluster or  removeFlinkCluster` if needed.

Reply:
> if the cluster is started successfully here, then we should add it into the monitor ?
> 
> In other scenerios with my limited read , `update`, `delete`,`shutdown` `start`, Its should be processed the logic by using FlinkClusterWatcher.addFlinkCluster or removeFlinkCluster` if needed.

This is my negligence. It is indeed necessary to remove it from the cache during `shutdown`, `start`.As for the `update`and `delete`scenarios, as the `flink cluster` is stopped, it means that there is no longer any `flink cluster` information in the cache, so I don't think it needs to be removed from the cache anymore.